portfolio/src/main/webapp/js/scripts.js

```diff
@@ -61,9 +61,24 @@ function createCommentElement(comment) {
     commentElement.className = 'comment';
 
     const textElement = document.createElement('span');
-    textElement.innerText = comment.text + " - " + comment.name;
+    textElement.innerText = comment.text + " - " + comment.name + "    ";
+
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'X';
+    deleteButtonElement.addEventListener('click', () => {
```

Comment:
Ah, this was the other method of adding onclick events we chatted about! (just a remark, no action required)